gst/tensor_decoder/tensordec.c

```diff
@@ -172,6 +172,22 @@ static gboolean gst_tensordec_transform_size (GstBaseTransform * trans,
     GstPadDirection direction, GstCaps * caps, gsize size,
     GstCaps * othercaps, gsize * othersize);
 
+/** tensor decoder image labeling mode data info  */
+TensorDec_Mode_image_Label *tensordec_mode_image_label;
```

Comment:
Because this is not exported, use ```static```.
And can you sure that there will be only ONE instance of tensordec_mode_image_label for the whole process? (Otherwise, you need to include this (has-a relation) in the corresponding tensor-dec object)

Reply:
yes, I will include this in tensor-dec object